setup.py

```diff
@@ -31,30 +31,31 @@
 
 kwargs = {
     'name': 'rocker',
-    'version': '0.2.3',
+    'version': '0.3.0',
```

Comment:
please don't change the version number in a patch. This will be done at release time.

Reply:
My point of view on this is that we add a functionality and thus must increment the minor version number but will obey to your request. I'll increment first RP #130.

Reply:
Yes, that's the policy for semantic versioning. But this commit is not the release. There may be multiple additions that go into the next release. And I couldn't merge this PR at the same time as your other one because this change would conflict.

Reply:
I get it with the version number. I actually never contributed to a release-based repository. How do you then manage the version change? Do contributors have to notify somewhere that they did a change that provokes a version change?